### DISCUSSION TITLE
xDrip reporting different than Dexcom

### DISCUSSION THREAD

**Discussion by todd3835** [NightscoutFoundation/xDrip#3423] [Bug reports]
Have a Dexcom G7 with xDrip in Companion mode. I just don't understand how it never matches what Dexcom shows. I've tried to wipe the app data with no luck. Any help would be appreciated.

[
![Screenshot_20240408-061039](https://github.com/NightscoutFoundation/xDrip/assets/6496147/dbe2d54e-0780-4ec8-800f-b7d234d0f037)
](url)

**Comment by Navid200** [comment]
Please post a screenshot of the Dexcom app so that it shows the smallest number of readings (3 hours)?
Then, do the same for xDrip for the same period so that we can see both.

**Reply by todd3835** [reply]
Dexcom app blocks screenshots. They just come out black

**Reply by Navid200** [reply]
Take a picture with a camera or another phone.
